packages/vats/src/vat-bank.js

```diff
@@ -40,7 +41,7 @@ const makePurseController = (
     async *getBalances(b) {
       assert.equal(b, brand);
       let updateRecord = await balanceNotifier.getUpdateSince();
-      while (updateRecord.updateCount) {
+      for await (const _ of whileTrue(() => !!updateRecord.updateCount)) {
```

Comment:
The shouting is already done in `whileTrue`, so I'd rather not shout louder.

```suggestion
      for await (const _ of whileTrue(() => updateRecord.updateCount)) {
```

Reply:
Agreed. That's what I tried first, but TS then shouted at me. Please advise.

Reply:
https://github.com/Agoric/agoric-sdk/pull/6744#pullrequestreview-1234088141

Reply:
Done